test/bigtest/network/factories/line.js

```diff
@@ -0,0 +1,17 @@
+import { Factory } from 'miragejs';
+import faker from 'faker';
+
+export default Factory.extend({
+  id: () => faker.random.uuid(),
+  purchaseOrderId: faker.random.uuid,
+  poLineNumber: (id) => `${id}${faker.random.alphaNumeric()}${faker.random.alphaNumeric()}${faker.random.alphaNumeric()}-${id}`,
```

Comment:
don't copy-paste weird code, just put some random string here

Reply:
Actually, maybe it's better to leverage poLine into stripes-acq-components, since it's used in ui-orders and ui-invoice as well

Reply:
created PR for [`stripes-acq-components`](https://github.com/folio-org/stripes-acq-components/pull/177)